packages/sdk-components-react/src/html-embed-patchers.ts

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
please don't add this globally, use it per line, otherwise its easy to forget where any was used unintentionally